### ISSUE TITLE
Tests can sometimes fail locally (even when passing in CI) because of the debug toolbar

### ISSUE THREAD

**Issue by louh** [usdoj-crt/crt-portal-management#1082]
## Summary

@ilodidoj noted (and I confirmed) that unit tests failed locally in https://github.com/usdoj-crt/crt-portal/pull/987, even though those tests passed in CI.

Upon closer investigation, I learned that local and CI tests behave differently because the [Django Debug Toolbar](https://github.com/jazzband/django-debug-toolbar) plugin is enabled locally, but not in other environments (such as CI). In some cases, the debug toolbar will contain strings of text that wouldn't otherwise appear in the output HTML. Tests that look for the absence of a particular string in the output HTML content will fail if that string appears in the debug toolbar markup, even if it correctly doesn't appear in other parts of the output.

### Steps To Reproduce

Run unit tests locally.

```sh
docker-compose run web python /code/crt_portal/manage.py test cts_forms --parallel
```

### Expected Behavior

Tests should behave the same in both local and CI environments. We should not be observing different test results in both environments.

### Actual Behavior

In this case, the test for `test_response_action_blank_submit` in `ResponseActionTests` failed this assertion:

```python
self.assertFalse('Contacted complainant:' in content)
```

The output HTML shows that the text string appears in debug content by the Django Debug Toolbar:

![image](https://user-images.githubusercontent.com/2553268/134715059-38d0f0a9-999c-4c2d-975a-3682d642b0a0.png)

(Ignore the extra `/n`, those are artifacts of the output and aren't the issue here)

This would appear to be introduced in https://github.com/usdoj-crt/crt-portal/commit/ae848e3aef708977722bc3b326bf9d323a6e2bcd#diff-b55042e384594dfb9eadeaa7e8f9ced49d6e93fa4d30a462910444b64142556fR398, which creates a new SQL query looking for the presence of the 'Contacted complainant:' string. This query is captured by the Django Debug Toolbar and becomes a part of the output HTML content, and the test fails.

## Suggested Actions

Since the debug toolbar doesn't make sense in an automated testing context, we should attempt to run tests without it. This way, existing tests that are currently testing HTML output won't behave unexpectedly across different environments.

Generally speaking, it's not a great idea to be testing string output on the entire HTML page; it's better to use something like a DOM targeting library to test output only where the unit tests affect a particular component or part of a page. However, that's a much bigger lift to address this situation when a much simpler method to guarantee local / CI environment parity exists.

We can [detect if we are in a test environment in Django](https://stackoverflow.com/questions/4088253/django-how-to-detect-test-environment-check-determine-if-tests-are-being-ru/7651002) and if so, set `ENABLE_DEBUG_TOOLBAR` to `False`. This will fix the test and also prevent test failures of a similar nature from occurring in the future.

## Acceptance Criteria

Local unit tests will pass, in particular, the test for `test_response_action_blank_submit` in `ResponseActionTests` that's currently failing.

**Comment by ilodidoj** [comment]
@louh I think we still need to look into this if this is related to pre-USWDS 2.8.x fix for this particular PR. This issue did not happen earlier and also should be on any PR created after post fix USWDS 2.8.x PR merge, which fix the send button issue in the modal.  I think the test is looking for that string match after the send button action.  Since send button is not working in this PR, the test failed.  That is my assumption, I might be wrong as well :(.  Key note, this test never failed before, and also work on most recently pulled dev version from the repo.

**Comment by louh** [comment]
@ilodidoj You're right that local tests passed before the USWDS 2.8.x introduction, but the test failure wasn't caused by that.

Local unit tests should pass on commit de3b85b8f7484b14c734f741d1c27ec7bb872ec4 (which includes all of USWDS 2.8.x), and it will fail on commit 3c500eb6950b44fef763fc167cfa836872243c82, which introduced the last-sent email on the total complaints activity table.
